hack/render-sync.sh

```diff
@@ -1,16 +1,35 @@
 #!/usr/bin/env bash
-
 WORKDIR=$(dirname "$(realpath "$0")")/..
-ARTIFACT_DIR=$(mktemp -d)
-
-cd "${WORKDIR}" || { echo "failed to change dir to ${WORKDIR}"; exit; }
-_output/cluster-node-tuning-operator render \
---asset-input-dir "${WORKDIR}"/test/e2e/performanceprofile/cluster-setup/manual-cluster/performance,"${WORKDIR}"/test/e2e/performanceprofile/cluster-setup/base/performance \
---asset-output-dir "${ARTIFACT_DIR}"
-
-cp "${ARTIFACT_DIR}"/*  "${WORKDIR}"/test/e2e/performanceprofile/testdata/render-expected-output/default
-for f in  "${WORKDIR}"/test/e2e/performanceprofile/testdata/render-expected-output/default/*
-do
-  sed -i "s/uid:.*/uid: \"\"/" "${f}"
-done
-rm -r "${ARTIFACT_DIR}"
+
+function join_by { local IFS="$1"; shift; echo "$*"; }
+
+function rendersync() {
+  INPUT_DIRS=()
+  while (( $# > 1 )); do
+    INPUT_DIRS+=("${WORKDIR}/test/e2e/performanceprofile/cluster-setup/$1")
+    shift
+  done
+  INPUT_DIRS=$(join_by , ${INPUT_DIRS[@]})
+
+  OUTPUT_DIR=$1
+
+  echo "== Rendering ${INPUT_DIRS} into ${OUTPUT_DIR}"
+
+  ARTIFACT_DIR=$(mktemp -d)
+
+  cd "${WORKDIR}" || { echo "failed to change dir to ${WORKDIR}"; exit; }
+  _output/cluster-node-tuning-operator render \
+  --asset-input-dir "${INPUT_DIRS}" \
+  --asset-output-dir "${ARTIFACT_DIR}"
+
+  cp "${ARTIFACT_DIR}"/*  "${WORKDIR}"/test/e2e/performanceprofile/testdata/render-expected-output/${OUTPUT_DIR}
+  for f in  "${WORKDIR}"/test/e2e/performanceprofile/testdata/render-expected-output/${OUTPUT_DIR}/*
+  do
+    sed -i "s/uid:.*/uid: \"\"/" "${f}"
+  done
```

Comment:
After extra thinking, this should probably be done once to reconcile the current state with the expected state, and then not enforced at every sync.
Other than this LGTM

Reply:
Extra: I'd remove the entry entirely rather than set to empty.

Reply:
I actually did not change this part, just moved it into a function.

But the sync is the reconciliation of current state with expected state. The uid reset only happend on make render-sync to make sure the tests themselves do not have to do that anymore.

IOW: I do not understand what do you mean by "once" here.